apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -201,6 +201,21 @@
       </div>
 
     </statusbar>
+
+    <portal to="modals">
+      <modal class="vue-dialog beekeeper-modal" :name="`discard-changes-modal-${tab.id}`">
```

Comment:
Unrelated: how do we generalize this stupid behavior of the modal library? It's super easy to make a mistake. Does it accept a `ref` instead of a `name`?

Reply:
I believe we've had this conversation before? I'll look into it, but I don't think it supports refs

Reply:
Maybe I'll look into filing an issue/PR on their repo cause we've run into this issue like 4 or 5 times now with the save modal, params modal, filter modal, and now this one lol